frontend/components/Pages/HelpAndSupportPage.tsx

```diff
@@ -12,8 +12,10 @@ import {
 } from '@/constants/urls';
 import { useElectronApi } from '@/hooks/useElectronApi';
 
-import { ArrowUpRightSvg, ExternalLink } from '../custom-icons';
+import { ArrowUpRightSvg } from '../custom-icons';
+import { ExternalLinkIcon } from '../custom-icons/ExternalLinkIcon';
 import { ExportLogsButton } from '../ExportLogsButton';
+import { cardStyles } from '../PearlWallet/Withdraw/common';
```

Comment:
I'd rather move it to some shared place than imported from Withdraw, this way we make a connection for two logically not connected pages